.github/workflows/application.yml

```
@@ -221,10 +226,17 @@ jobs:
                     key: ${{ runner.os }}-php-${{ matrix.php }}-symfony-${{ matrix.symfony }}-composer-${{ hashFiles('**/composer.json') }}
                     restore-keys: |
                         ${{ runner.os }}-php-${{ matrix.php }}-symfony-${{ matrix.symfony }}-composer-
+
             -   name: Install PHP dependencies
+                if: matrix.php != '8.2'
                 run: composer update --no-interaction --no-scripts
                 id: end-of-setup
 
+            -   name: Install PHP 8.2 dependencies
+                if: matrix.php == '8.2'
+                run: composer update --no-interaction --no-scripts --ignore-platform-req=php
```

To allow updates of upper requirement only
```suggestion
                run: composer update --no-interaction --no-scripts --ignore-platform-req=php+
```

---

Why do you want to drop 8.0? 
- We still support it
- We run tests in parallel anyway
- On OSS projects GHA is free
🤔